api/setting.go

```diff
@@ -37,6 +37,8 @@ func (s *Server) SettingsGet(w http.ResponseWriter, req *http.Request) error {
 		value = si.DefaultEngineImage
 	case types.SettingEngineUpgradeImage:
 		value = si.EngineUpgradeImage
+	case types.SettingSecretConfig:
```

Comment:
The name `SecretConfig` is too ambiguous. Use `BackupTargetCredentialSecret` instead.